src/ia-anniversary-banner.ts

```diff
@@ -129,7 +133,7 @@ export class IaAnniversaryBanner extends LitElement {
         <a
           href=${link}
           alt="Come celebrate 25 years with us."
-          data-event-click-tracking="Anniv25Banner|Homepage"
+          @click=${this.bannerClick}
```

Comment:
Q: Do you need to track close button click events too?

Reply:
yep, there is already one in for close button ^_^